### DISCUSSION TITLE
Add the ability to switch between accounts

### DISCUSSION THREAD

**Discussion by JustMaier** [civitai/civitai#66] [Ideas]
It'd be cool to have a way to quickly toggle between multiple accounts

**Comment by JTJutajoh** [comment]
Bumping this because I was going to post a similar request.

For various reasons, often privacy-related, it might be in some people's interests to have two accounts for the website for posting different types of content. Currently, the login flow of clicking a link in an email makes it prohibitively time consuming to switch between two different accounts, it would be nice if there was some interface (inspired by google) to swap between multiple accounts. The current login flow also makes it very annoying to log in through an incognito window, since it means you have to log in to your email in the incognito window as well.

On the upside, I imagine it would make it fairly easy for you to track which accounts belong to the same person so you can factor that in to reduce the impact of rating manipulation using multiple accounts.
